hookman/hookman_generator.py

```diff
@@ -447,7 +447,7 @@ def _plugin_cmake_file_content(self, shared_lib_name):
               set(CMAKE_C_FLAGS_DEBUG "-g")
             endif(NOT WIN32)
 
-            set(CMAKE_CXX_FLAGS       "-Wall -Werror=return-type -ftemplate-depth=1024")
+            set(CMAKE_CXX_FLAGS       "-Wall")
```

Comment:
I think this could be removed from WIN32 build as well, MSVC generates too many useless warnings

Reply:
- Commit https://github.com/ESSS/hookman/pull/12/commits/9c8d963118769c32760b0d38f5afa1fcddf72d3b fix this issue ;)